utils_test.py

```diff
@@ -16,6 +16,7 @@
 
 # pylint: disable=relative-import
```

Comment:
Updating it to following gets rid of all errors:
`from oppia.core.tests import test_utils
import oppia.feconf as feconf
import oppia.utils as utils
`